packages/obonode/obojobo-chunks-figure/editor-component.test.js

```diff
@@ -1,17 +1,23 @@
+import Figure from './editor-component'
+import ModalUtil from 'obojobo-document-engine/src/scripts/common/util/modal-util'
 import React from 'react'
 import { mount } from 'enzyme'
 import renderer from 'react-test-renderer'
-
-import Figure from './editor-component'
-
-import ModalUtil from 'obojobo-document-engine/src/scripts/common/util/modal-util'
+import EditorStore from 'obojobo-document-engine/src/scripts/oboeditor/stores/editor-store'
```

Comment:
Unfortunately had to add direct access to the store here to get at the upload media settings